bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/java/org/openhab/binding/bluetooth/daikinmadoka/internal/model/commands/GetOperationHoursCommand.java

```diff
@@ -0,0 +1,115 @@
+import org.openhab.binding.bluetooth.daikinmadoka.internal.model.MadokaValue;
+import org.openhab.core.library.types.QuantityType;
+import org.openhab.core.util.HexUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This command returns the operating hours of internal unit
+ *
+ * @author Benjamin Lafois - Initial contribution
+ *
+ */
+@NonNullByDefault
+public class GetOperationHoursCommand extends BRC1HCommand {
+
+    private final Logger logger = LoggerFactory.getLogger(GetOperationHoursCommand.class);
+
+    private @Nullable QuantityType<Time> indoorOperationHours;
+    private @Nullable QuantityType<Time> indoorFanHours;
+    private @Nullable QuantityType<Time> indoorPowerHours;
+
+    @Override
+    public byte[][] getRequest() {
+        MadokaValue specificUnitNumber = new MadokaValue(0x02, 1, new byte[] { (byte) 0x00 });
+        MadokaValue p40 = new MadokaValue(0x40, 0, new byte[] {});
+        MadokaValue p41 = new MadokaValue(0x41, 0, new byte[] {});
+        MadokaValue p42 = new MadokaValue(0x42, 0, new byte[] {});
+        MadokaValue p43 = new MadokaValue(0x43, 0, new byte[] {});
+        MadokaValue p44 = new MadokaValue(0x44, 0, new byte[] {});
+        MadokaValue p45 = new MadokaValue(0x45, 0, new byte[] {});
+        MadokaValue p46 = new MadokaValue(0x46, 0, new byte[] {});
+        MadokaValue p47 = new MadokaValue(0x47, 0, new byte[] {});
+        MadokaValue p48 = new MadokaValue(0x48, 0, new byte[] {});
+
+        return MadokaMessage.createRequest(this, specificUnitNumber, p40, p41, p42, p43, p44, p45, p46, p47, p48);
+    }
+
+    @Override
+    public void handleResponse(Executor executor, ResponseListener listener, MadokaMessage mm)
+            throws MadokaParsingException {
+        try {
+
+            byte[] msg = mm.getRawMessage();
+            if (logger.isDebugEnabled() && msg != null) {
+                logger.debug("Got response for {} : {}", this.getClass().getSimpleName(), HexUtils.bytesToHex(msg));
+            }
+
+            // The specific GetOperationHours requires 2 consecutive runs for some reason.
+            // If value size is 0, then it will be for the next query!
+            if (mm.getValues().get(0x40).getSize() == 0) {
+                setState(State.SUCCEEDED);
+                return;
+            }
+
+            Integer iIndoorOperationHours = (int) (mm.getValues().get(0x40).getComputedValue(ByteOrder.LITTLE_ENDIAN));
+            Integer iIndoorFanHours = (int) (mm.getValues().get(0x41).getComputedValue(ByteOrder.LITTLE_ENDIAN));
+            Integer iIndoorPowerHours = (int) (mm.getValues().get(0x42).getComputedValue(ByteOrder.LITTLE_ENDIAN));
+
+            this.indoorOperationHours = new QuantityType<Time>(iIndoorOperationHours,
+                    org.openhab.core.library.unit.Units.HOUR);
+            this.indoorFanHours = new QuantityType<Time>(iIndoorFanHours, org.openhab.core.library.unit.Units.HOUR);
```

Comment:
Please just import the core package instead of declaring here.
```suggestion
            this.indoorFanHours = new QuantityType<Time>(iIndoorFanHours, Units.HOUR);
```

Please update wherever applicable.